desmume/src/frontend/interface/interface.cpp

```diff
@@ -39,6 +39,7 @@
 
 #define SCREENS_PIXEL_SIZE 98304
 volatile bool execute = false;
+volatile bool rom_opened = false;
```

Comment:
i don't think this needs to be volatile, unless EXPORTED can be declared as static. otherwise there's an interface barrier using the extern function and the compiler can't optimize out the access to the variable.

Reply:
Brushed up my knowledge about volatile and static (I'm really not much of a C/C++ dev...) and yeah, I shouldn't just have copy-pasted that. I changed it to static.